cookbook/routing/extra_information.rst

```diff
@@ -52,12 +52,16 @@ arguments to your controller:
 
         return $collection;
 
-Now, you can access this extra parameter in your controller::
+Now, you can access this extra parameter in your controller, either as an argument (if specified), or through the ``Request`` object::
```

Comment:
I would remove the addition here and change the code to the example proposed by @javiereguiluz.